developer/acceptance-testing/scripts/order-creation.md

```diff
@@ -0,0 +1,112 @@
+ 3. Click the "Authorize" & "Capture" checkboxes for the supported methods.
+ 4. Select "Test - Sanbox Mode" from the Mode selectbox.
+ 5. Enter the API Login ID & the Transaction Key.
+ 6. Click the "save changes" button.
+ 
+**Braintree**
+ 1. Click the payment option in the admin menu to open the payment provider settings panel.
+ 2. Click the checkbox to right of the Braintree label to open the Braintree settings.
+ 3. Enter a Merchant ID, Public Key and Private Key.
+ 4. Clcik the "Authorize", "Capture" & "Refund" checkboxes for the supported methods.
+ 5. Select "Testing - Sandbox Mode" from the Mode selectbox.
+ 6. Click the "save changes" button.
+
+**PayPal Express**
+ 1. Click the payment option in the admin menu to open the payment provider settings panel.
+ 2. Click the checkbox to right of the PayPal Express label to open the PayPal Express settings.
+ 3. Enter a Merchant ID, Username, Password and Signature.
+ 4. Clcik the "Authorize", "Capture" & "Refund" checkboxes for the supported methods.
+ 5. Select "Authorize, Capture on complete" from the Capture at time of auth selectbox.
+ 6. Select "Testing - Sandbox Mode" from Express Mode selectbox.
+ 7. Click the "save changes" button.
+ 
+**PayPal Flow**
+ 1. Click the payment option in the admin menu to open the payment provider settings panel.
+ 2. Click the checkbox to right of the PayPal Flow label to open the PayPal Flow settings.
+ 3. Eneter an API Client ID and a API Secret.
+ 4. Clcik the "Authorize", "Capture" & "Refund" checkboxes for the supported methods.
+ 5. Select "Testing - Sandbox Mode" from the Payflow Mode selectbox.
+ 6. Click the "save changes" button.
+ 
+**Stripe**
+ 1. Click the payment option in the admin menu to open the payment provider settings panel.
+ 2. Click the checkbox to right of the Stripe label to open the Stripe settings.
+ 3. Eneter an API Secret Key.
+ 4. Clcik the "Authorize", "Capture" & "Refund" checkboxes for the supported methods.
+ 5. Click the "save changes" button.
+
+### Shipping
+**Flat Rate**
+ 1. Click the shipping option in the admin menu to open the shipping settings panel.
+ 2. Click the checkbox to right of the Flat Rate label to open the Flat Rate settings.
+  . Click the "save changes" button.
+ 
+**Shippo**
+ 1. Click the shipping option in the admin menu to open the shipping settings panel.
+ 2. Click the checkbox to right of the Shippo label to open the Shippo settings.
+ 3. Click the "save changes" button.
+
+### Taxation
+**Custom Rates**
+ 1. Click the taxation option in the admin menu to open the taxation settings panel.
+ 2. Click the checkbox to right of the Custom Rates label to open the Custom Rates settings.
+ 3. Click the "save changes" button.
+
+**Avalara**
+ 1. Click the taxation option in the admin menu to open the taxation settings panel.
+ 2. Click the checkbox to right of the Avalara label to open the Avalara settings.
```

Comment:
If they aren't using the stock settings, there is a bunch of stuff they need to configure for this to work correctly.